.github/workflows/check-merge-conflict.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge Conflict Check
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check if prs are dirty
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        with:
+          dirtyLabel: has-conflicts
```

Comment:
Should this fit in our existing label schema? I'm thinking sth. like `status:conflicts`. @LukasKalbertodt, you are the label master. Opinions?

Reply:
Label master :D 

But yes, `status:conflicts` sounds good. I just created that label.

Reply:
fine with me. Will use that